docs/json_data/sysfs_attributes_drm.json

```
@@ -539,6 +539,138 @@
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": "GPU_CORE_FREQUENCY_STATUS"
+        },
+        "MAX_FREQ": {
+            "attribute": "freq0/max_freq",
+            "scalar": 1e6,
+            "description": "Maximum GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": true,
+            "alias": "GPU_CORE_FREQUENCY_MAX_CONTROL"
+        },
+        "MIN_FREQ": {
+            "attribute": "freq0/min_freq",
+            "scalar": 1e6,
+            "description": "Minimum GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": true,
+            "alias": "GPU_CORE_FREQUENCY_MIN_CONTROL"
+        },
+        "RP0_FREQ": {
+            "attribute": "freq0/rp0_freq",
+            "scalar": 1e6,
+            "description": "Maximum non-overclocked GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": "GPU_CORE_FREQUENCY_MAX_AVAIL"
+        },
+        "RPA_FREQ": {
+            "attribute": "freq0/rpa_freq",
+            "scalar": 1e6,
+            "description": "Actual GPU RPA frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": ""
+        },
+        "RPE_FREQ": {
+            "attribute": "freq0/rpe_freq",
+            "scalar": 1e6,
+            "description": "GPU RPE frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": ""
+        },
+        "RPN_FREQ": {
+            "attribute": "freq0/rpn_freq",
+            "scalar": 1e6,
+            "description": "Minimum requestable GPU frequency",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
+            "format": "double",
+            "units": "hertz",
+            "writeable": false,
+            "alias": "GPU_CORE_FREQUENCY_MIN_AVAIL"
+        },
+        "IDLE_RESIDENCY": {
+            "attribute": "gtidle/idle_residency_ms",
+            "scalar": 1e-3,
+            "description": "Time spent in idle state",
+            "doc_domain": "gpu_chip",
+            "aggregation": "average",
+            "behavior": "variable",
```

```suggestion
            "behavior": "monotone",
```
Sounds like an incrementing counter. Have you come across docs/implementation/comments to confirm?

Or if it is variable, then I'd guess we need to update the description to "Time spent in idle state `<per or since> <???>`"